src/AccessibilityInsights.Extensions.GitHubAutoUpdate/ProductionChannelInfoProvider.cs

```diff
@@ -28,7 +28,7 @@ public ProductionChannelInfoProvider(IGitHubWrapper gitHubWrapper, IExceptionRep
         /// </summary>
         public bool TryGetChannelInfo(ReleaseChannel releaseChannel, out ChannelInfo channelInfo)
         {
-            return ChannelInfoUtilities.TryGetChannelInfo(releaseChannel, out channelInfo, _gitHubWrapper, null, _exceptionReporter);
```

Comment:
>null [](start = 107, length = 4)

Passing null here was a bug. We want to use the default parameter, hence the named parameter syntax